### DISCUSSION TITLE
Enzyme Failing to build

### DISCUSSION THREAD

**Discussion by EthanHarness** [EnzymeAD/Enzyme#1364] [Q&A]
Hello! I've been trying to setup enzyme for a project I'm working on, and I have been having some trouble building it. I believe I followed the installation instructions correctly. The only thing that I did differently was instead of building in a folder called build I changed it to buildEnzyme since there is an already existing file called BUILD in that folder. When building the project I get the following error 

![image](https://github.com/EnzymeAD/Enzyme/assets/87549399/194e1b31-a32d-4df4-b3be-8b9dbc896658)

Any help from the community would be appreciated!

**Comment by ZuseZ4** [comment]
Looks like you are on Windows? Are you using WSL, or can you give some details on your setup and the commands used please?

**Reply by EthanHarness** [reply]
Sure. I'm on Windows 11. I'm not using WSL. 

Started in the directory C:\Users\OWNER\source\repos
git clone --config core.autocrlf=false https://github.com/llvm/llvm-project.git
cd llvm-project
mkdir build
cd build
cmake -G Ninja ../llvm -DLLVM_TARGETS_TO_BUILD="host" -DLLVM_ENABLE_PROJECTS="clang" -DLLVM_ENABLE_PLUGINS=ON -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON
ninja

(Went back to the directory C:\Users\OWNER\source\repos)
git clone https://github.com/EnzymeAD/Enzyme
cd Enzyme
cd enzyme
mkdir buildEnzyme
cd buildEnzyme
cmake -G Ninja .. -DLLVM_DIR=C:\Users\OWNER\source\repos\llvm-project\build\lib\cmake\llvm
ninja (Error ocurrs)

**Reply by scaevolabars** [reply]
Hi, I came across with similar problem. Deleting that file and creating directory with name `build` helped
